spec/controllers/api/order_cycles_controller_spec.rb

```diff
@@ -0,0 +1,158 @@
+require "spec_helper"
+
+module Api
+  describe OrderCyclesController, type: :controller do
```

Comment:
Ideally, this would have been included in the commit adding the controller.

Reply:
Ideally, yes. The controller changed several times through the PR, I added the spec once it was stable...